### PR TITLE
feat: minor ui fixes and updates

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**
- Fixes the width of carousel cards
- Fixes loader to animate clockwise
- Update `.env.example` to mention required variables
- Added commitlint to streamline commit messages (Follows conventional commit messages spec)
- Move redundant functions to utils
- Fixes typos of `milliseconds` and tutorialContainer

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       }
     ],
     "paths": {
-      "@/*": ["./*"]
+      "@chomp/*": ["./*"]
```
Comment (AnteVuletic) [inline tsconfig.json:21]: I would prefer if we did not merge this

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
@adithyaakrishna thanks for this. But this is a big commit with some opinionated changes we would need to check internally. Could you just include the .env.example changes for now?

File: app/actions/deck/factories.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const deckInputFactory = (
         ({
           question: question.question,
           type: question.type,
-          durationMiliseconds: ONE_MINUTE_IN_MILISECONDS,
+          durationMilliseconds: ONE_MINUTE_IN_MILISECONDS,
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply app/actions/deck/factories.ts:34]: Oops! My bad, I missed this one. Thanks for pointing it out. Will update it :)

**Comment by adithyaakrishna** [comment]
@marvinmarnold Gotcha, should I keep the typo ones or revert them as well?

**Pull request closed without merging**